draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -406,6 +406,36 @@ On the server side, this mostly helps avoid resource exhaustion. For clients, pr
 
 The value of the idle timeout to use depends on the exact deployment and is a trade-of between resource usage on clients/servers and the overhead of opening new connections. Very short timeouts that are at or below the timeouts used for application layer watchdogs, typically in the range of 30-60s can be considered unreasonable. In contrast, the upper limit is much more difficult to define but may be in the range of 10 to 15min, depending on the available resources, or never (disabling idle timeout) in scenarios where a permanently open connection is required.
 
+## Malformed Packets and Unknown clients
+
+The RADIUS specifications say that an implementation should "silently discard" a packet in a number of circumstances.
+This action has no further consequences for UDP based transports, as the "next" packet is completely independent of the previous one.
+
+When TLS is used as transport, decoding the "next" packet on a connection depends on the proper decoding of the previous packet.
+As a result the behavior with respect to discarded packets has to change. With DTLS, while the "next" packet does not depend on proper decoding of the previous packet, but since DTLS provides
+integrity protection, any decoding error should be treated as a broken connection.
```

Comment:
```suggestion
As a result the behavior with respect to discarded packets has to change, since a malformed RADIUS packet could impact the decoding of succeeding packets.

With DTLS, the "next" packet does not depend on proper decoding of the previous packet, since the RADIUS packets are sent in independent DTLS records.
However, since both TLS and DTLS provide integrity protection and ensure that the packet was sent by the peer, a protocol violation at this stage implies that the peer is misbehaving.
```